resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/PVTestsSuite.scala

```diff
@@ -210,6 +210,7 @@ private[spark] trait PVTestsSuite { k8sSuite: KubernetesSuite =>
         CONTAINER_MOUNT_PATH)
       .set(s"spark.kubernetes.executor.volumes.persistentVolumeClaim.data.options.claimName",
         PVC_NAME)
+      .set(s"spark.executor.instances", "2")
```

Comment:
Sorry but I would disagree with this test case because the underlying is not `NFS`.

Reply:
I'm trying to see if it's possible to set up NFS in k8s integration testing.